boxoffice/forms/order.py

```diff
@@ -55,3 +55,14 @@ class OrderSessionForm(forms.Form):
     utm_id = forms.StringField(__("UTM Id"), filters=[trim(250)])
     gclid = forms.StringField(__("Gclid"), filters=[trim(250)])
     referrer = forms.StringField(__("Referrer"), filters=[trim(2083)])
+
+
+class OrderRefundForm(forms.Form):
+    amount = forms.IntegerField(__("Amount"),
+        validators=[forms.validators.DataRequired(__("Please specify an amount"))])
+    internal_note = forms.TextAreaField(__("Internal note"),
+        description=__("Add a note for future reference"), filters=[forms.filters.none_if_empty()])
+    refund_description = forms.TextAreaField(__("Refund description"),
+        description=__("Why is this order receiving a refund?"), filters=[forms.filters.none_if_empty()])
```

Comment:
A textarea field works (for the sake of rendering long strings) as long as we strip newlines from it. Hasjob does this in the jobpost reject form, for instance.